components/Table.tsx

```diff
@@ -162,7 +162,9 @@ const columns = [
           textAlign: "right",
         }}
       >
-        {Number.isNaN(row.tvl24hChange) ? "-" : `${row.tvl24hChange}%`}
+        {Number.isNaN(row.tvl24hChange) || row.tvl24hChange === null
```

Comment:
this might be undefined and not null, but 0 is a valid value.  you may want to try the `??` operator https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
I actually don't think that's appropriate here as we aren't returning the value directly. But it's a fair point.

If you think it could be undefined, I'll cover that edge case.